sunpy/net/tests/strategies.py

```diff
@@ -15,6 +16,11 @@
 from sunpy.net import attrs as a
 from sunpy.time import TimeRange, parse_time
 
+if sys.version_info >= (3, 12):
+    now_year = datetime.datetime.now(datetime.UTC).year
```

Comment:
Does this not work with Pytbon 3.9? The docs suggest there's support for tzinfo on `now()`: https://docs.python.org/3.9/library/datetime.html#datetime.datetime.now

Reply:
I will adjust